roles/glance/templates/etc/glance/policy.json

```diff
@@ -7,7 +7,7 @@
     "get_image": "",
     "get_images": "",
     "modify_image": "",
-    "publicize_image": "role:admin",
+    "publicize_image": ["role:admin", "role:cloud_admin"],
```

Comment:
this should be 
"role:admin or role:cloud_admin" using the new format for policy files.